cmd/azmount/filemanager/azure.go

```diff
@@ -30,19 +30,19 @@ func tokenRefresher(credential azblob.TokenCredential) (t time.Duration) {
 	// JWT tokens comprise three fields. the second field is the payload (or claims).
 	// we care about the `aud` attribute of the payload
 	curentTokenFields := strings.Split(currentToken, ".")
-	logrus.Infof("current token fields: %v", curentTokenFields)
+	logrus.Infof("Current token fields: %v", curentTokenFields)
 
 	payload, err := base64.RawURLEncoding.DecodeString(curentTokenFields[1])
 	if err != nil {
-		logrus.Errorf("Error decoding base64 token payload: %v", err)
+		logrus.Errorf("Error decoding base64 token payload: %s", err)
 		return 0
 	}
-	logrus.Infof("current token payload: %v", string(payload))
+	logrus.Infof("Current token payload: %s", string(payload))
```

Comment:
Is there anything sensitive in this token?  Probably should move to debug if there's risk.